boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi

```diff
@@ -51,6 +51,34 @@
 	};
 };
 
+&clk_msi{
```

Comment:
These whitespaces are not very consistent, so I think we'll need a cleaning pass after this mass conversion. @erwango, do we have something like checkpatch for dts files?

Reply:
We do (dtc), but unfortunately, it doesn't catch this.